applications/ExaquteSandboxApplication/tests/cpp_tests/test_metric_process.cpp

```diff
@@ -0,0 +1,366 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Vicente Mataix Ferrandiz
+//                   Riccardo Tosi
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "geometries/triangle_2d_3.h"
```

Comment:
Using Element2D3 and Element3D4 you don't need this

Reply:
Done, thanks for this hint!